twilio_integration/twilio_integration/doctype/whatsapp_message/whatsapp_message.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2021, Frappe and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe.model.document import Document
+from six import string_types
+from frappe.utils.password import get_decrypted_password
+from frappe import _
+from twilio_integration.twilio_integration.doctype.twilio_settings.twilio_settings import get_twilio_client
+
+class WhatsAppMessage(Document):
+	def send(self):
+		client = get_twilio_client()
+		message_dict = self.get_message_dict()
+		response = frappe._dict()
+
+		try:
+			response = client.messages.create(**message_dict)
+			self.sent_received = 'Sent'
+			self.status = response.status.title()
+			self.id = response.sid
+			self.send_on = response.date_sent
+			self.save(ignore_permissions=True)
+		
+		except Exception as e:
+			self.db_set('status', "Error")
+			frappe.log_error(e, title = _('Twilio WhatsApp Message Error'))
+	
+	def get_message_dict(self):
+		args = {
+			'from_': self.from_,
+			'to': self.to,
+			'body': self.message,
+			'status_callback': 'https://01d3954808ab.ngrok.io/api/method/twilio_integration.twilio_integration.api.whatsapp_message_status'
+		}
+		if self.media_link:
+			args['media_url'] = [self.media_link]
+
+		return args
+
+def send_bulk_whatsapp_message(receiver_list, message, doctype, docname, media=None):
```

Comment:
Can we move this into whatapp message class and make it as a class method?

Reply:
Then how do I import the class method if needed(in the notification).